packages/image_picker/image_picker_for_web/lib/image_resizer.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math';
+import 'dart:ui';
+import 'package:image_picker_for_web/image_resizer_utils.dart';
+import 'package:image_picker_platform_interface/image_picker_platform_interface.dart';
+import 'dart:html' as html;
+
+/// Helper class that resizes image
+class ImageResizer {
```

Comment:
Please, move this image_resizer and its utils to `lib/src`, so people can't `import 'package:image_picker_for_web/image_resizer.dart';` from outside of this package.

Reply:
done